00XX:reassert-fee-payment-method

```diff
@@ -0,0 +1,54 @@
+## HIP: Driving Value Into The IOT Network Through IOT Token Payments
+- Author: @maxgold91 
+- Start Date: 2023-04-17
+- Category: Technical
+- Status: Draft
+- Original PR:
+- Tracking Issue:
+- Vote Requirements: 
+
+## Summary
+
+This HIP proposes changing the method of payment for Location Assert Fees from HNT tokens to IOT tokens. The primary purpose of this change is to drive value into IOT tokens and provide them with a small amount of utility. This HIP does not propose altering any other aspect of Location Assert Fees. The proposed changes are intended to be implemented in compliance with the rules set forth in [HIP-51](https://github.com/helium/HIP/blob/main/0051-helium-dao.md).
+
+## Motivation
+
+Currently, Location Assert Fees are paid using HNT tokens. However, there is no incentive to pay for onboarding using HNT under HIP-51, as none of the burned tokens accrue back to the subDAO. This proposal seeks to address this issue by transitioning the method of payment to IOT tokens.
+
+By adopting IOT tokens as the method of payment, we aim to drive value into the IOT ecosystem and give IOT tokens a small amount of utility. Moreover, this change aligns with the rules established in HIP-51, as detailed below.
+
+## Specification
+
+### Method of Payment
+
+This HIP proposes that Location Assert Fees be paid using IOT tokens instead of HNT tokens. The change will be effected by modifying the relevant code within the Helium network core implementation. This modification will ensure that only IOT tokens are accepted for Location Assert Fees.
```

Comment:
"location assert fees on the IOT Network".

I would just specify again that this only impacts the IOT network.